packs/linux/actions/dig.py

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/python
+
+import subprocess, random, re
+from st2actions.runners.pythonrunner import Action
+
+class DigAction(Action):
+
+    def run(self,rand,count,nameserver,hostname,queryopts):
+        opt_list = []
+        output = []
+        nameserver = '@' + nameserver
+        cmd_args = ['dig', nameserver]
+
+        if re.search(',',queryopts):
```

Comment:
In this case you don't really need to use regular expression, you could just do ``if ',' in queryopts`.